### PR TITLE
Added quizzes

### PR DESCRIPTION
**Pull request by onkabonka92**

### CODE REVIEW

**Comment by sandris-** [comment]
there is no need to create new files, use existing ones

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
-# CODELEX 04-2019
+# Quiz
```
Comment (sandris-) [inline README.md]: why are you changing this file?

File: arithmetic.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+```csharp
```
Comment (sandris-) [inline arithmetic.md]: each question starts with ---

File: README.md (hunk 1 above)
Comment (onkabonka92) [reply README.md]: To be honest I dont know how did it happen

**Pull request closed without merging**